examples/aggregator/main_module/tests/integration_test.rs

```diff
@@ -14,26 +14,43 @@
 // limitations under the License.
 //
 
-use aggregator::{data::SparseVector, handler::SAMPLE_THRESHOLD};
 use aggregator_common::Monoid;
 use aggregator_grpc::proto::{aggregator_client::AggregatorClient, Sample, SerializedSparseVector};
+use aggregator_handler::{data::SparseVector, SAMPLE_THRESHOLD};
+use anyhow::Context;
 use assert_matches::assert_matches;
 use maplit::hashmap;
 use oak_abi::{
     label::{confidentiality_label, web_assembly_module_tag},
     proto::oak::application::ConfigMap,
 };
-use oak_sign::get_sha256;
 use std::{
     collections::HashMap,
     convert::{From, TryFrom},
 };
 
-const WASM_MODULE_FILE_NAME: &str = "aggregator.wasm";
-const WASM_MODULE_MANIFEST: &str = "../../module/rust/Cargo.toml";
-const MODULE_NAME: &str = "app";
+const MAIN_MODULE_FILE_NAME: &str = "aggregator.wasm";
+const HANDLER_MODULE_FILE_NAME: &str = "aggregator_handler.wasm";
+
+const MAIN_MODULE_MANIFEST: &str = "../main_module/Cargo.toml";
+const HANDLER_MODULE_MANIFEST: &str = "../handler_module/Cargo.toml";
+
+const MAIN_MODULE_NAME: &str = "app";
+const HANDLER_MODULE_NAME: &str = "handler";
 const ENTRYPOINT_NAME: &str = "oak_main";
 
+// SHA-256 hash of Handler module.
+// TODO(865): Remove and compute SHA-256 hash once reproducibility is fixed.
+const HANDLER_SHA256_HASH: &str =
+    "93539428d76d71d1812267013880b0aafcb40cbcc42b49d5dca511422a33690f";
```

Comment:
Do we have a test with the wrong hash label?

Reply:
I just have sent it to test if CI for aggregator produces the same hash.
But it looks like it does.
I think we need to fix reproducibility before submitting this PR